cookie.cabal

```diff
@@ -1,5 +1,5 @@
 name:            cookie
-version:         0.4.2.1
+version:         0.4.2.2
```

Comment:
Sorry, I should have clarified. Since this is adding a new identifier, this needs to be a minor version bump, not a patch. Meaning: the new version number would be 0.4.3.

Reply:
Ah yes sorry, I misread the PVP spec. Updated.